html/css/dark.css

```diff
@@ -1,3 +1,11 @@
+.ie9 img {
+    width: 100%;
+}
+@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+    img {
+        width: 100%;
```

Comment:
Yeah, I did some quick searches and tried a few things and could not fix the logos on IE11.
Tried different combinations of viewbox and height/width svg properties and the css here.